tensorflow/contrib/tensorrt/trt_conversion.i

```diff
@@ -0,0 +1,98 @@
+/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+/* Wrap trt_conversion */
```

Comment:
TODO: Fix the swig problem. I added this comment to prevent other TF authors from accidentally merging this PR.

Reply:
@aaroey, I fixed the swig interface problems but now we see difference between converted and native model. Something else seem to be broken as well. Investigating.